relay-server/src/actors/store.rs

```diff
@@ -664,8 +664,8 @@ impl StoreService {
         // message because we can achieve better parallelism when dealing with a single
         // message.
 
-        // Max message size is 1MB.
-        let max_message_size = 1000 * 1000;
+        // Max message size is 15MB.
+        let max_message_size = 1000 * 150000;
```

Comment:
Shouldn't this be:
```suggestion
        let max_message_size = 1000 * 15000;
```

if you want to have 15mb?

Reply:
Could we also introduce the configuration option for this and not hardcode it?